src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Queries/GetApprenticeships/GetApprenticeshipsQueryResult.cs

```diff
@@ -24,9 +24,10 @@ public class ApprenticeshipDetails
             public long ProviderId { get; set; }
             public string CourseName { get; set; }
             public DeliveryModel DeliveryModel { get; set; }
-            public DateTime StartDate { get; set; }
+            public DateTime StartDate { get; set; }            
             public DateTime EndDate { get; set; }
             public DateTime PauseDate { get; set; }
+            public DateTime StopDate { get; set; }
```

Comment:
This should be a nullable value

Reply:
PauseDate is also often null in the database - ive followed the same pattern